packages/analytics-js-integrations/__tests__/integrations/TikTokAds/tiktokads.test.js

```diff
@@ -375,6 +378,7 @@ describe('TiktokAds Track event', () => {
       partner_name: 'RudderStack',
       contents: [
         {
+          brand: '',
```

Comment:
why blank?

Reply:
not any particular reason just changed the output based upon input vals

Reply:
Better to choose non blank value